### PR TITLE
[DOCFIX] Add MapR-FS documentation

### PR DESCRIPTION
**Pull request by gpang**

### CODE REVIEW

File: docs/en/Configuring-Alluxio-with-MapR-FS.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  <td>2.4.1-mapr-1408</td>
+</tr>
+</table>
+
+# Configuring Alluxio for MapR-FS
+
+Once you have compiled Alluxio with the appropriate `hadoop.version` for your MapR distribution, you have to configure
+Alluxio to recognize the MapR-FS scheme and URIs. Alluxio can use the HDFS client to access MapR-FS. In order to enable
+the HDFS client to access MapR-FS URIs, you have to add the URI prefix `maprfs:///` to the configuration variable
+`alluxio.underfs.hdfs.prefixes`.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-MapR-FS.md]: Maybe mention which file it should be changed in?

File: docs/en/Configuring-Alluxio-with-MapR-FS.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+```
+alluxio.underfs.hdfs.prefixes=hdfs://,glusterfs:///,maprfs:///
+```
+
+This configuration parameter should be set for all the Alluxio servers (masters, workers). Please read how to
+[configure Alluxio](Configuration-Settings.html).
+
+This parameter should be set whenever you access Alluxio. This means the parameter should be set for any application
+(MapReduce, Spark, Flink, etc.) that accesses Alluxio.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-MapR-FS.md]: Perhaps combine this paragraph with the previous and add some instruction about how to set it for the clients?

File: docs/en/Configuring-Alluxio-with-MapR-FS.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+# Configuring Alluxio to use MapR-FS as Under File System
+
+There are various ways to configure Alluxio to use MapR-FS as the Under File System. If you want to mount MapR-FS to the
+root of Alluxio, add the following to `alluxio-env.sh`:
+ 
+```
+ALLUXIO_UNDERFS_ADDRESS=maprfs:///<path in MapR-FS>/
+```
+
+You can also mount MapR-FS to the Alluxio namespace.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-MapR-FS.md]: `to a folder in`?

**Comment by gpang** [comment]
@calvinjia thanks for the comments! I updated the PR.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**